lib/modules/dosomething/dosomething_mbp/dosomething_mbp.install

```diff
@@ -144,6 +144,14 @@ function dosomething_mbp_update_7008() {
 }
 
 /**
+ * Updates mbp productions to include messagingGroupsQueue.
+ */
+function dosomething_mbp_update_7009() {
+  db_drop_table('message_broker_producer_productions');
```

Comment:
Unfortunately, that's normal way to update the mapping. See updates 7004, 7005, 7007, 7008.

That's another one example of a legacy code. It supposed to be refactored to features, but, I believe, it'll be deprecated before the refactoring.